packages/cbor/package.json

```diff
@@ -23,7 +23,7 @@
     "release": "npm version patch && git push --follow-tags && npm publish",
     "predev": "npm run coverage",
     "dev": "light-server -q -s. -w 'lib/*.js,test/*.js # npm run coverage' -o /coverage/lcov-report/index.html",
-    "types": "tsc"
+    "types": "tsc && (cp -r ./types ../cbor-web/ || xcopy types ..\cbor-web /E)"
```

Comment:
Can you use the copyfiles package or something like it please, so that we're using the same code on both OS's?

Reply:
yea surely

Reply:
@hildjj how about if I create a simple .js script and put these commands in it?

Reply:
@hildjj I updated the command using your suggested lib